examples/rust/src/bin/client.rs

```diff
@@ -549,6 +550,23 @@ async fn geyser_subscribe(
                         );
                         continue;
                     }
+                    Some(UpdateOneof::Ping(_)) => {
+                        // This is necessary to keep load balancers that expect client pings alive. If your load balancer doesn't
+                        // require periodic client pings then this is unnecessary
+                        subscribe_tx
+                            .send(SubscribeRequest {
+                                ping: Some(SubscribeRequestPing { id: 1 }),
+                                slots: request.slots.clone(),
+                                accounts: request.accounts.clone(),
+                                transactions: request.transactions.clone(),
+                                entry: request.entry.clone(),
+                                blocks: request.blocks.clone(),
+                                blocks_meta: request.blocks_meta.clone(),
+                                commitment: request.commitment.clone(),
+                                accounts_data_slice: request.accounts_data_slice.clone(),
```

Comment:
```suggestion
                                ping: Some(SubscribeRequestPing { id: 1 }),
                                ..Default::default()
```
if `ping` field is defined all other fields are ignored, so we can set them to default value

Reply:
Oh right I was testing on a branch that didn't have the latest ping changes

Reply:
Updated, sorry for delay

Reply:
np, thanks for the update!